### ISSUE TITLE
Search interface in Wide

### ISSUE THREAD

**Issue by chandramouleswaran** [chandramouleswaran/Wide#50]
Need a good search functionality embedded in the Window header similar to VS.
![Screenshot!](http://www.codesmiles.com/image.axd?picture=Visual%20Studio%202012-QuickLaunch.jpg)

**Comment by ghost** [comment]
i think u need an autocomplete for the search box?
currently playing arround with this one:
http://intellibox.codeplex.com/
except for the long loading one, all examples are running

**Comment by joazlazer** [comment]
Yeah maybe use some events from the text box for when text changes.

**Comment by ghost** [comment]
? or not? O.o

**Comment by joazlazer** [comment]
Oh it is all implemented already. However, you would need to change it for the 'metro feel.' Maybe since mah apps metro already styles pop ups, it may be a better idea to implement your own.

**Comment by ghost** [comment]
yes, just the result display must be changed...
